README.md

```diff
@@ -34,7 +34,165 @@ NNI automates feature engineering, neural architecture search, hyperparameter tu
 
 <img src="docs/img/overview.svg" width="100%"/>
 
-(TBD: table)
+<table>
+<tbody>
+<tr align="center" valign="bottom">
+<td></td>
+<td>
+<b>Hyperparamter Tuning</b>
+<img src="docs/img/bar.png" />
+</td>
+<td>
+<b>Neural Architecture Search</b>
+<img src="docs/img/bar.png" />
+</td>
+<td>
+<b>Model Compression</b>
+<img src="docs/img/bar.png" />
+</td>
+</tr>
+<tr valign="top">
+<td align="center" valign="middle">
+<b>Algorithms</b>
+</td>
+<td>
+<ul>
+<li><b>Exhaustive search</b></li>
+<ul>
+<li><a href="https://nni.readthedocs.io/en/doc-refactor/reference/hpo.html#nni.algorithms.hpo.gridsearch_tuner.GridSearchTuner">Grid Search</a></li>
```

Comment:
not doc-refactor

Reply:
Should we use latest or stable? 🤔

Reply:
Not sure. I used stable for other links in readme. But master might contain un-released features. In that case, we won't have stable.

Reply:
I meant, stable are dead links until next release.